src/components/Builder/Achievement/index.jsx

```diff
@@ -216,8 +244,8 @@ const Achievement = ({ achievementData }) => {
                     rules={[
                       {
                         required: true,
-                        message: "Name is required",
-                      },
+                        message: "Name is required"
```

Comment:
I think you guys are using different eslink or some settings or commas in the last getting added and sometimes getting removed. Please check together why this is happening.

Reply:
Done!